test/http2_frame_size_SUITE.erl

```diff
@@ -34,14 +34,15 @@ wrong_size_window_update(Config) ->
 
 send_wrong_size(Type, _Config) ->
     {ok, Client} = http2c:start_link(),
-    http2c:send_binary(Client, <<10:24,Type:8,0:1,0:31,0:100>>),
+    http2c:send_binary(Client, <<10:24,Type:8,0:1,0:31,0:104>>),
```

Comment:
hahaha.

Dialyzer caught this. 100 was generating ending with a bit size that was not evenly dividing into 8.